source_gen/lib/src/generator_for_annotation.dart

```diff
@@ -42,12 +42,25 @@ import 'type_checker.dart';
 /// may be helpful to check which elements have a given annotation.
 abstract class GeneratorForAnnotation<T> extends Generator {
   final bool throwOnUnresolved;
+  final String? inPackage;
```

Comment:
Doc comments here, please? So it's clear how they are used.

Reply:
Done.